anydex/wallet/iota/iota_provider.py

```diff
@@ -0,0 +1,125 @@
+from datetime import datetime
+
+from iota import AsyncIota, ProposedTransaction, Seed
+
+from anydex.wallet.provider import Provider
+
+
+class IotaProvider(Provider):
```

Comment:
Wouldn't it be better to have an abstract provider for Iota and then inherit from it to create this Provider?

Reply:
I could work on that, so that other providers could be implemented down the line with more ease